packages/core/src/modules/vc/W3cCredentialService.ts

```diff
@@ -336,23 +336,27 @@ export class W3cCredentialService {
     return w3cCredentialRecord
   }
 
-  public async getAllCredentials(): Promise<W3cVerifiableCredential[]> {
-    const allRecords = await this.w3cCredentialRepository.getAll()
-    return allRecords.map((record) => record.credential)
+  public async removeCredentialRecord(id: string) {
+    const cred = await this.w3cCredentialRepository.getById(id)
+    await this.w3cCredentialRepository.delete(cred)
   }
 
-  public async getCredentialById(id: string): Promise<W3cVerifiableCredential> {
-    return (await this.w3cCredentialRepository.getById(id)).credential
+  public async getAllCredentialRecords(): Promise<W3cCredentialRecord[]> {
```

Comment:
We don't use this in all other services/modules (just getAll instead of getAllCredentials, getAllCredentialRecords). Do you think this helps? If so we should probably change this in all methods

Reply:
I get your point in terms of consistency. However, I consciously  did this because the `W3cCredentialService` deals with more than credential records. For example, there is a `signPresentation()` method and a `getSignatureSuitesForCredential()`. It seems a bit odd to me to specify the data model in some method names, but leave it out in others.

I'm fine with changing it though. However, in that case it would make sense to me to rename  `W3cCredentialService` to `W3cCredentialRecordService`. Then at least the class name gives you a hint of what `getAll()` actually returns.